tests/js/spec/components/search/sources/apiSource.spec.jsx

```diff
@@ -279,7 +279,7 @@ describe('ApiSource', function () {
     });
 
     // There are no members that match
-    expect(mock.mock.calls[1][0].results).toHaveLength(5);
```

Comment:
Not quite sure what's going on here yet. Looking into why these tests need to change 🤔

Reply:
Looks like this is here because of fuzzy search. `foo` also finds Octohook.

Reply:
Might be nice to add a comment re: fuzzy search